EVENT.md

```diff
@@ -143,17 +149,20 @@ window.fetch("https://adtech.example/attribution_source?my_ad_id=123",
              { headers });
 ```
 
-The `<a>`, `<img>`, and `window.open` mechanisms will cause the browser to
-initiate a `keepalive` fetch request to the URL indicated by `attributionsrc`.
+Specifying a URL value for `attributionsrc` within `<a>`, `<img>`, or
```

Comment:
It might make sense to pull up this section before discussing the fetch API, since `attributionsrc` stuff does not apply in that case.

Reply:
Updated PTAL.